Automattic/jetpack
# CRM: Adding a Back To List button on many edit and add new pages

## Proposed changes:

* This PR adds a 'Back to List' button  to the top of the Edit Quote page. It also adds it to the Add New Quote page, to match the Add New and Edit Contact pages.
* Similarly, it does the same for the Add New and Edit Transaction, Invoice and Company pages (added later)

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2944

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this, use the Jetpack Beta plugin on a test site with this branch applied or test locally with the branch checked out.
* Visit the 'Add New' Quote page (`/wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=quote`), and make sure the Back to List button shows. Make sure the rest of the menu matches the screenshot below, and the links all work properly.
* Visit the 'Edit Quote' page for any existing quote (`/wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=quote&zbsid={quoteid}`), and make sure the Back to List button shows. Make sure the rest of the menu matches the screenshot below, and the links all work properly.
* Visit the same two pages on trunk (or using the release candidate / bleeding edge if using the Beta plugin), and make sure the top navigation is the same as before minus the Back to List button.
* As well as as Quotes, check the Transactions, Invoices and Companies pages as well. The only difference should be the Back to List button.

Screenshots (just for Quotes, as an example):

**Before**

Add New Quote page:

<img width="1719" alt="Screenshot 2023-03-23 at 10 20 47" src="https://user-images.githubusercontent.com/16754605/227173900-bde0c87c-d23e-4734-a06a-b91292f17f7b.png">

Edit Quote page:

<img width="1732" alt="Screenshot 2023-03-23 at 10 21 31" src="https://user-images.githubusercontent.com/16754605/227174024-c66a46b1-cc52-421b-9a17-97a231811215.png">



**After**

Add New Quote page:
<img width="1730" alt="Screenshot 2023-03-23 at 10 20 06" src="https://user-images.githubusercontent.com/16754605/227173741-943c330a-7a5d-40b4-944b-742570e59a4c.png">


Edit Quote page:

<img width="1731" alt="Screenshot 2023-03-23 at 10 19 43" src="https://user-images.githubusercontent.com/16754605/227173575-97d39708-65aa-4461-a207-39dd316d0815.png">
